BlueprintUI/Sources/Internal/Extensions/CGRect.swift

```diff
@@ -24,6 +24,20 @@ extension CGRect {
         )
     }
 
+    /// Creates a new rectangle by rounding the origin and size of this rectangle.
+    /// - Parameters:
+    ///   - rule: The rounding rule.
+    ///   - scale: The rounding scale.
+    /// - Returns: A rectangle with the rounded values.
+    func roundedForSize(_ rule: FloatingPointRoundingRule, by scale: CGFloat) -> CGRect {
```

Comment:
Minor since this is an internal method, but the `roundedForFrame` vs `roundedForSize` are kind of confusing to me – eg reading the names I don't exactly know how they differ. But I also don't have a better suggestion!

Reply:
(Maybe we should fold these into one method that takes in the `ViewDescription.RoundingPriority` and switches on that?)

Reply:
Ha that's exactly what I went for in my prototype, with "size" and "border" cases. Agree the current names are a bit confusing

Reply:
Double edit: looked at my code and I had:

```swift

public enum PixelRoundingPriority {
    case edges
    case size
}
```

I think edges is much clearer than frame.

Reply:
Yeah edges is better, that's what I wrote in the docs too. 👍